editor/project_settings_editor.cpp

```diff
@@ -484,4 +479,14 @@ ProjectSettingsEditor::ProjectSettingsEditor(EditorData *p_data) {
 
 	get_ok_button()->set_text(TTR("Close"));
 	set_hide_on_ok(true);
+
+	inspector->set_restrict_to_basic_settings(true);
+	if (ProjectSettings::get_singleton()->has_setting("_project_settings_use_advanced_mode")) {
+		bool use_advanced = ProjectSettings::get_singleton()->get("_project_settings_use_advanced_mode");
+		if (use_advanced) {
+			inspector->set_restrict_to_basic_settings(false);
+			advanced->set_pressed(true);
```

Comment:
If you select advanced mode and then reopen the PS, it will show advanced settings but not the advanced bar. This is because the PS hides the advanced bar by default (see L280, NOTIFICATION_VISIBILITY_CHANGED).

Reply:
good catch